src/components/DateRangePicker.jsx

```diff
@@ -112,6 +112,7 @@ const defaultProps = {
 
   // internationalization
   displayFormat: () => moment.localeData().longDateFormat('L'),
+  inputFormats: [moment.localeData().longDateFormat('L')],
```

Comment:
Instead of this new prop, could we overload `displayFormat` so that it returned either an array or a string? 

I believe that not much else in the code would have to change.

Reply:
And if it's an array, we could default to the first format. I wonder if that's too brittle. 🤔

Reply:
@majapw Any news?